### ISSUE TITLE
Foundry v10 Testing 5 (Build 279): Run QuickEncounter doesn't work on a Journal Page

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#101]
```
QuickEncounter.js:290 Uncaught (in promise) TypeError: Cannot set properties of null (setting 'showQEOnce')
    at QuickEncounter.serializeIntoJournalEntry (QuickEncounter.js:290:35)
    at QuickEncounter.update (QuickEncounter.js:318:14)
    at QESheet._onChange (QESheet.js:267:26)
    at QESheet._updateObject (QESheet.js:244:18)
    at QESheet._onSubmit (foundry.js:5771:18)
    at QESheet.submit (foundry.js:6090:16)
    at HTMLButtonElement.<anonymous> (QESheet.js:116:22)
    at HTMLButtonElement.dispatch (jquery.min.js:2:43064)
    at v.handle (jquery.min.js:2:41048)
```

I believe this is happening because `this.journalId` is set but journalEntry is not

**Comment by spetzel2020** [comment]
With the fixes from 1.0.4j, you now get the "There is no Map Note" and the error has now moved to:
```
EncounterNote.js:193 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'contains')
    at EncounterNote.place (EncounterNote.js:193:33)
    at yes (EncounterNote.js:221:31)
    at callback (foundry.js:54862:35)
    at button.callback (foundry.js:54913:51)
    at Dialog.submit (foundry.js:54800:35)
    at Dialog._onClickButton (foundry.js:54730:10)
    at HTMLButtonElement.dispatch (jquery.min.js:2:43064)
    at v.handle (jquery.min.js:2:41048)
```

There are two problems:
1. Even if there is a Map Note attached to the Journal Entry it is not being found because we are looking at the Journal Entry Page
2. If there isn't a Map Note then the above error is relevant

**Comment by spetzel2020** [comment]
Converted to use parent JE throughout if we were on a JournalEntryPage
Fixed in 1.0.4k
Released in 1.0.4
